server/src/main/resources/rules/rules.js

```diff
@@ -502,53 +506,41 @@ function get_pool_priority(pool, consumer) {
         var null_rule_score = 0;
         var mismatch_rule_score = 0;
 
+        log.debug("Evaluating attribute " + attr + " with weight " + attrs[attr]);
```

Comment:
Even though this is in the Javascript, the logger still supports some of the string building/formatting placeholders we use in Java. I think there may be a parameter limit or something to that effect, but for the most part, that syntax can (and probably should) be used.

Reply:
Changed the log statements to use string formatting